Prace_domowe/Praca_domowa2/Jakub_Lis/pd2.ipynb

```diff
@@ -0,0 +1,753 @@
+{
```

Comment:
<p>Bardzo dobrze. Taka drobna uwaga, zamiast <code>np.sqrt(mean_squared_error(...))</code> mogłeś użyć <code>mean_squared_error(..., squared=False)</code>. Oczywiście nie ma to wpływu na wynik. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>